mar/tiebout-action.hoon

```diff
@@ -0,0 +1,40 @@
+::
+::::  /hoon/actions/collection/mar
+  ::
+/?  309
+/-  hall
+/+  tiebout
+=,  tiebout
+=,  format
+::
+|_  act=action:tiebout
+::
+++  grow
+  |%
+  ++  tank  >act<
+  --
+::
+++  grab
+  |%
+  ++  noun  action:tiebout
+  ++  json
+    |=  jon=^json
+    ^-  action:tiebout
+    %-  (hard action:tiebout)
+    =<  (action jon)
+    |%
+    ++  action
+      %-  of:dejs
+        :~  token+so:dejs
+            add-circle+sa
+            del-circle+sa
+        ==
+    ++  sa            :: string as ta
+      |=  jon=^json
+      ?>  ?=([%s *] jon)
+      (scot %tas p.jon)
```

Comment:
If you want a `@tas` from the `@t` `p.jon`, you can either just cast it, or `?>  (sane %tas p.jon)` and then cast, or similar. `(scot %tas` just gives you the "rendered" version, which may not be what you want.